src/server/server.cc

```diff
@@ -152,6 +152,19 @@ Status Server::Start() {
     }
   }
 
+  std::string value;
```

Comment:
We'd better clarify this behavior in the configuration file:

https://github.com/apache/kvrocks/blob/9d656f1476583651e2fae20ed1c37cf35d0679ca/kvrocks.conf#L46-L48